exporter/signalfxexporter/testdata/json/hostmetrics.json

```diff
@@ -0,0 +1,20 @@
":"0"},{"attributes":[{"key":"device","value":{"stringValue":"awdl0"}},{"key":"direction","value":{"stringValue":"receive"}}],"startTimeUnixNano":"1659645944000000000","timeUnixNano":"1660944008800763000","asInt":"0"},{"attributes":[{"key":"device","value":{"stringValue":"bridge0"}},{"key":"direction","value":{"stringValue":"transmit"}}],"startTimeUnixNano":"1659645944000000000","timeUnixNano":"1660944008800763000","asInt":"0"},{"attributes":[{"key":"device","value":{"stringValue":"bridge0"}},{"key":"direction","value":{"stringValue":"receive"}}],"startTimeUnixNano":"1659645944000000000","timeUnixNano":"1660944008800763000","asInt":"0"},{"attributes":[{"key":"device","value":{"stringValue":"llw0"}},{"key":"direction","value":{"stringValue":"transmit"}}],"startTimeUnixNano":"1659645944000000000","timeUnixNano":"1660944008800763000","asInt":"0"},{"attributes":[{"key":"device","value":{"stringValue":"llw0"}},{"key":"direction","value":{"stringValue":"receive"}}],"startTimeUnixNano":"1659645944000000000","timeUnixNano":"1660944008800763000","asInt":"0"},{"attributes":[{"key":"device","value":{"stringValue":"en1"}},{"key":"direction","value":{"stringValue":"transmit"}}],"startTimeUnixNano":"1659645944000000000","timeUnixNano":"1660944008800763000","asInt":"0"},{"attributes":[{"key":"device","value":{"stringValue":"en1"}},{"key":"direction","value":{"stringValue":"receive"}}],"startTimeUnixNano":"1659645944000000000","timeUnixNano":"1660944008800763000","asInt":"0"},{"attributes":[{"key":"device","value":{"stringValue":"en2"}},{"key":"direction","value":{"stringValue":"transmit"}}],"startTimeUnixNano":"1659645944000000000","timeUnixNano":"1660944008800763000","asInt":"0"},{"attributes":[{"key":"device","value":{"stringValue":"en2"}},{"key":"direction","value":{"stringValue":"receive"}}],"startTimeUnixNano":"1659645944000000000","timeUnixNano":"1660944008800763000","asInt":"0"},{"attributes":[{"key":"device","value":{"stringValue":"en3"}},{"key":"direction","value":{"stringValue":"transmit"}}],"startTimeUnixNano":"1659645944000000000","timeUnixNano":"1660944008800763000","asInt":"0"},{"attributes":[{"key":"device","value":{"stringValue":"en3"}},{"key":"direction","value":{"stringValue":"receive"}}],"startTimeUnixNano":"1659645944000000000","timeUnixNano":"1660944008800763000","asInt":"0"},{"attributes":[{"key":"device","value":{"stringValue":"en4"}},{"key":"direction","value":{"stringValue":"transmit"}}],"startTimeUnixNano":"1659645944000000000","timeUnixNano":"1660944008800763000","asInt":"0"},{"attributes":[{"key":"device","value":{"stringValue":"en4"}},{"key":"direction","value":{"stringValue":"receive"}}],"startTimeUnixNano":"1659645944000000000","timeUnixNano":"1660944008800763000","asInt":"0"},{"attributes":[{"key":"device","value":{"stringValue":"utun1"}},{"key":"direction","value":{"stringValue":"transmit"}}],"startTimeUnixNano":"1659645944000000000","timeUnixNano":"1660944008800763000","asInt":"0"},{"attributes":[{"key":"device","value":{"strin
```

Comment:
shouldn't be only one line? I believe only one is being unmarshaled anyway. Please confirm

Reply:
You're right! I removed all of the other lines and the metric data and benchmark test results were the same.

I'll update.